### PR TITLE
`data : bhavanas-jpg`

### PR DESCRIPTION
**Pull request by bhavanas-jpg**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3602"></a>

### CODE REVIEW

File: data/bhavanas-jpg.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Bhavana S",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend developer | Blogger | Poetry & Literature lover| Yoga Practitioner",
+  "avatar": "https://pbs.twimg.com/profile_images/1607335763548725249/KDIKS-wp_400x400.jpg",
```
Comment (AbbeyIT) [inline data/bhavanas-jpg.json:6]: You can copy your GitHub profile URL and just add `.png` at the end.

**Review by AbbeyIT — CHANGES_REQUESTED** [review]

File: data/bhavanas-jpg.json (hunk 1 above)
Comment (Pradumnasaraf) [reply data/bhavanas-jpg.json:6]: Thank you for the suggestion, Abbey. People can have a custom link. Thank you.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

File: data/bhavanas-jpg.json (hunk 1 above)
Comment (AbbeyIT) [reply data/bhavanas-jpg.json:6]: Thanks for this Pradumna. Gonna take note for future PRs.

**Pull request merged**